block_types/block_base.py

```diff
@@ -12,11 +14,9 @@ def compile_lines(self, func, lines):
 		try:
 			func.compile_blocks(lines)
 		except CompileError as e:
-			print(e)
-			raise CompileError(f'Unable to compile {self.block_name} at line {self.line}')
-		except e:
-			print(traceback.format_exc())
```

Comment:
Why are we getting rid of printing tracebacks? Having tracebacks are invaluable for debugging.

Reply:
Correct, sorry , I was thinking about the end user experience however , which suffers from them.